test/integration/smoke/test_ssvm.py

```diff
@@ -75,7 +75,7 @@ def tearDown(self):
         return
 
     def waitForSystemVMAgent(self, vmname):
-        timeout = self.services["timeout"]
+        timeout = 120
```

Comment:
I've heavily refactored and fixed this test in my branch/PR #2211. In general, throughout the test codebase we should NOT be sleeping blindly but use `wait_until` and poll for behaviour (i.e list stuff) over some short period (1-10 seconds) and retries.

Reply:
Sure, let's leave them be.